build/containers-scenarios.yml

```
@@ -40,6 +40,8 @@ parameters:
       arguments: --variable rate=1000 --property rate=1000 --load.job bombardier
     - displayName: 10000
       arguments: --variable rate=10000 --property rate=10000 --load.job bombardier
+    - displayName: 20000
+      arguments: --variable rate=20000 --property rate=20000 --load.job bombardier
```

```suggestion
    - displayName: 20000
      arguments: --variable rate=20000 --property rate=20000 --load.job bombardier
```

Let's remove it for now and add it if it's asked for. That would double the number of runs, and for fortunes it's already at 50% (and more) of CPU with 10K.